src/mixins/filing-template-mixin.ts

```diff
@@ -146,7 +146,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
         correctedFilingType: this.getCorrectedFilingType,
         correctedFilingDate: this.getCurrentDate,
         comment: `${this.defaultCorrectionDetailComment}\n${this.getDetailComment}`,
-        type: this.getClientErrorCorrection
+        type: this.getCorrectionType
```

Comment:
I think you need to parse this correction error type from a draft filing (probably in `parseCorrectionFiling()`).